### ISSUE TITLE
example sentence for 3.2.5: Change on Request already covered by 3.2.2: On Input

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1779]
https://www.w3.org/WAI/WCAG22/Understanding/change-on-request

3.2.5: Change on Request mentions:

> This Success Criterion aims to eliminate potential confusion that may be caused by unexpected changes of context such as automatic launching of new windows, automatic submission of forms after selecting an item from a list, etcetera.

Now the second example given:

> automatic submission of forms after selecting an item from a list

Is already covered by 3.2.2: On Input, not?

https://www.w3.org/WAI/WCAG22/Understanding/on-input

So as an example to use as a reason for this extra SC while already covered seems to loose weight.

**Comment by jake-abma** [comment]
3.2.5 is not even used as an example at the before / after demo:

Nr. 3

https://www.w3.org/WAI/demos/bad/before/annotated/template.html

> `<select onchange="...">`
> Note: Errors on the template are propagated throughout the entire website.
> 
> Success Criterion 2.4.1 - Keyboard
> Failure 54: Failure of Success Criterion 2.1.1 due to using only pointing-device-specific event handlers (including gesture) for a function
> 
> Success Criterion 3.2.2 - On Input
> Failure 36: Failure of Success Criterion 3.2.2 due to automatically submitting a form and presenting new content without prior warning when the last field in the form is given a value

**Comment by mraccess77** [comment]
On a related note - 3.2.1 and 3.2.2 do not officially cover the situation where a dialog randomly pops up on page shifting focus as they are very scoped to on focus and on input.  However, all most everyone would flag this as a violation of SC 3.2.1 although technically it falls under SC 3.2.5.  This has been a long standing messy relationship between the two.  Also new windows that open on activation but without wording is crammed into SC 3.2.5 as well complicating things - some people continue to flag this as an issue under SC 3.2.1 which is not correct as indicating that links open new windows is not covered in Level A or AA.

**Comment by jake-abma** [comment]
@mraccess77 yeh, and change on input does not cover inputs, but also placed under 3.2.5, I think this is not correct and must be part of 3.2.2, also from a user perspective (not being AAA) but also from a 'doable perspective'

**Comment by alastc** [comment]
So the AAA criterion is more expansive than the AA one, I'm not seeing an issue.

**Comment by jake-abma** [comment]
the issue is we do not need to repeat ourselves at a AAA when the AA already covers it, AAA must be an 'addition' to the AA, not a repeat of the AA. Makes it more polluted, more cognitive load, harder to explain to people, and serves no need for why the AAA exists in the first place

**Comment by alastc** [comment]
But the AA doesn't cover it (all), the AAA has a wider scope. See the the recent issue on whether onBlur should be added to 3.2.1 as an example.

Just because the AAA SC has an example (to help explain it) that overlaps with the AA version is not, in itself, a problem.

If someone would like to come up with an example that highlights the difference between them, great, but it doesn't show an issue with the SCs or how they are explained individually.

**Comment by patrickhlauke** [comment]
as part of some little digging for https://github.com/w3c/wcag/pull/4955#issuecomment-3969076265 I think I may have hit on an initial disambiguation. I'd say 3.2.5 and 3.2.2 do need to acknowledge their overlap, but also their difference (for 3.2.2 you can get away with warning the user, for 3.2.5 that's not enough and a mechanism needs to be available to suppress the change of context). will work on a PR at some point for this...
